Fix an issue on AWS-GuardDuty content Pack when no findings are fetched (#32309)
demisto/content: Packs/AWS-GuardDuty/ReleaseNotes/1_3_44.md

#### Integrations
##### AWS - GuardDuty

> - Fix an issue on AWS-GuardDuty content Pack when no findings are fetched.

Reviewer: ```suggestion
- Fix an issue where the integration would fail when no findings are fetched.
```
